config/application.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'boot'
+
+require "rails"
+require "active_model/railtie"
+require "active_job/railtie"
+require "active_record/railtie"
+require "active_storage/engine"
+require "action_controller/railtie"
+require "action_mailer/railtie"
+require "action_mailbox/engine"
+require "action_text/engine"
+require "action_view/railtie"
+require "action_cable/engine"
+
+Bundler.require(*Rails.groups)
+
+module MovieSelectorApiJune2020
+  class Application < Rails::Application
+    config.load_defaults 6.0
+    config.api_only = true
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins '*'
+        resource '*', 
+          headers: :any, 
+          methods: %i[get post put delete],
+          expose: %w(access-token expiry token-type uid client),
+          max_age: 0
+      end
+  end
+end
```

Comment:
It's an end missing, make sure to format correctly and close every do/class/module (don't copy paste)